src/Main.cc

```diff
@@ -491,11 +491,13 @@ static void showSplash()
                  ord::OpenRoad::getGitDescribe());
   logger->report(
       "Features included (+) or not (-): "
-      "{}Charts {}GPU {}GUI {}Python",
+      "{}Charts {}GPU {}GUI {}Python{}",
       ord::OpenRoad::getChartsCompileOption() ? "+" : "-",
       ord::OpenRoad::getGPUCompileOption() ? "+" : "-",
       ord::OpenRoad::getGUICompileOption() ? "+" : "-",
-      ord::OpenRoad::getPythonCompileOption() ? "+" : "-");
+      ord::OpenRoad::getPythonCompileOption() ? "+" : "-",
+      strcmp(BUILD_TYPE, "Release") == 0 ? ""
```

Comment:
following the "+" / "-" convention above, absent features are not represented by a blank string, because then it is hard to tell what is missing.

Similarly, using a blank string for release, makes it hard to tell that there should have been a check here.